app/src/components/attachments/AttachmentsList.tsx

```diff
@@ -63,6 +75,12 @@ const AttachmentsList: React.FC<IAttachmentsListProps> = (props) => {
 
   const [currentAttachment, setCurrentAttachment] = useState<IGetProjectAttachment | IGetSurveyAttachment | null>(null);
 
+  useEffect(() => {
+    props.getAttachments(true);
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [surveyContext.surveyDataLoader]);
+
```

Comment:
There may be some opportunity (now or later) to update this component's props. Many of them seem old/outdated. For example, the checkbox/selected attachments props dont seem to be used anywhere. Probably added in past and never utilized.

Similarly, now that the survey context contains the attachments dataloader, there is probably a better way to leverage it (here or in the parent), so we don't have to pass down a custom `getAttachments` function.

I don't have an active ticket now that my PR is open, so I could maybe look at this tomorrow (or we can just make a ticket about it for next sprint).